testsuite/python/ek_charged_plate.py

```diff
@@ -164,11 +164,18 @@ def test(self):
                 negative_ions[i, j, 30].density = 0.0
 
         # Test error when trying to change ekin parameters after initialisation
-        ek._params.update({'agrid': 3,
-                           'T': 0.01})
         with self.assertRaises(RuntimeError):
+            ek._params.update({'agrid': 3, 'T': 0.01})
             ek._set_params_in_es_core()
 
+        # Check errors from the constructor
+        with self.assertRaisesRegex(ValueError, r"The following keys have to be given as keyword arguments: "
```

Comment:
should D not pop up in the 'have to be given' list?

Reply:
`r"\[.+\]"` means any sequence of characters, which includes D.

Reply:
oh ok